cosmwasm/contracts/wormhole/src/state.rs

```diff
@@ -238,7 +238,7 @@ impl GuardianSetInfo {
         if self.addresses.is_empty() {
             return 0;
         }
-        ((self.addresses.len() * 10 / 3) * 2) / 10 + 1
+        ((self.addresses.len() * 2) / 3) + 1
```

Comment:
Do each of the chains we're modifying quorum calculation here have independant quorum unit-tests to ensure that this doesn't cause a regression?  I'm thinking of what exists for Solidity / Guardians:

- https://github.com/wormhole-foundation/wormhole/blob/dev.v2/node/pkg/processor/quorum_test.go
- https://github.com/wormhole-foundation/wormhole/blob/dev.v2/ethereum/forge-test/Messages.t.sol#L13-L29

Where possible, I'd love to see the core trust assumptions, like quorum, well unit-tested on all chains.  This is especially true when making medications to those implementations to help prevent regression scenarios.

Reply:
That said, the math is pretty straight forward, so as a function of the review, I agree with the implementation changes, would just like to see associated test coverage.